docs/src/installation/index.md

```diff
@@ -116,7 +116,7 @@ $ sudo port install tectonic
 If you're using [nix], you can imperatively install [`tectonic`][nix-tectonic] with:
 
 ```sh
-$ nix-env -iA nixos.tectonic
+$ nix-env -iA nixpkg.tectonic
```

Comment:
Should be `nixpkgs`. Perhaps better is:

```shell
$ nix-env -f '<nixpkgs>' -iA tectonic
```

Reply:
I'm afraid that I have no expertise here. @matu3ba does the above change look good to you?

Reply:
Yes, it sounds better.